compiler/src/codegen/compcore.re

```diff
@@ -1969,7 +1973,7 @@ let allocate_array_n = (wasm_mod, env, num_elts, elt) => {
           compiled_num_elts(),
           Expression.const(wasm_mod, encoded_const_int32(0)),
         ),
-        InvalidArgument,
+        IndexOutOfBounds,
```

Comment:
For these two, it didn't seem worth it to me to set up calling an exception with arguments in the compiler, when this is just going to be rewritten into Grain soon anyway. I feel that IndexOutOfBounds is also an appropriate error here until this is moved over. If anyone really cares, I can do it sooner.